oak-store-document/src/test/java/org/apache/jackrabbit/oak/plugins/document/BranchTest.java

```diff
@@ -56,6 +56,12 @@ public class BranchTest {
     @Rule
     public DocumentMKBuilderProvider builderProvider = new DocumentMKBuilderProvider();
 
+    @Before
+    public void before() {
```

Comment:
```suggestion
    @After
    public void after() {
```

Shouldn't this be in an after method rather than before?

Reply:
(or there could be before and after, but only a before leaves state changed for other upcoming unit tests)

Reply:
Done. I put it in after method.